### PR TITLE
Updated to use SonarQube

### PR DESCRIPTION
**Pull request by tomasz-kaminski-sonarsource**

### CODE REVIEW

**Comment by tomasz-kaminski-sonarsource** [comment]
Pipelines can be found here: https://gitlab.com/tomasz.kaminski/linux-cmake-gitlab-ci-sq/-/pipelines

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 image: gcc
 
 variables:
-  SONAR_SERVER_URL: "https://sonarcloud.io"
```
Comment (amelie-renard-sonarsource) [inline .gitlab-ci.yml:4]: You also need to update the link on line 6

File: README.adoc [hunk 2]
```diff
@@ -29,12 +20,12 @@ It is very easy to analyze a C/C++/Objective-C project with SonarCloud https://s
 . Make sure that you have your token stored as a CI variable in your project (`SONAR_TOKEN`). You can request new tokens using https://sonarcloud.io/account/security/[Account/Security page].
```
Comment (amelie-renard-sonarsource) [inline README.adoc]:
```suggestion
. Make sure that you have your token stored as a CI variable in your project (`SONAR_TOKEN`). You can https://docs.sonarqube.org/latest/user-guide/user-token/[generate and use] new tokens in SonarQube.
```

File: .gitlab-ci.yml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 image: gcc
 
 variables:
-  SONAR_SERVER_URL: "https://sonarcloud.io" 
+  SONAR_SERVER_URL: <SONARQUBE_HOST_URL> # Replace <SONARQUBE_HOST_URL> with your SonarQube server URL
```
Comment (amelie-renard-sonarsource) [inline .gitlab-ci.yml:4]: And replace `sonarcloud-check` with `sonarqube-check` on line 56

**Review by amelie-renard-sonarsource — CHANGES_REQUESTED** [review]

File: .gitlab-ci.yml (hunk 1 above)
Comment (tomasz-kaminski-sonarsource) [reply .gitlab-ci.yml:4]: Thanks! Done.

**Review by amelie-renard-sonarsource — APPROVED** [review]

**Pull request merged**